tests/test_strategy_load_hours.py

```diff
@@ -497,12 +504,22 @@ def test_predefined_load_strategy_rejects_incorrect_rate_parameters(use_mmr, ini
                             energy_rate_increase_per_update=-1)
 
 
-def test_load_hour_strategy_increases_rate_when_fit_to_limit_is_false():
+def test_load_hour_strategy_increases_rate_when_fit_to_limit_is_false(market_test1):
     load = LoadHoursStrategy(avg_power_W=100, initial_buying_rate=0, final_buying_rate=30,
                              fit_to_limit=False, energy_rate_increase_per_update=10,
                              update_interval=5)
     load.area = FakeArea()
+    load.owner = load.area
     load.event_activate()
+    assert load.state._energy_requirement_Wh[TIME] == 25.0
+    offer = Offer('id', now(), 1, (MIN_BUY_ENERGY/500), 'A', market_test1)
+    load._one_sided_market_event_tick(market_test1, offer)
+    assert load.bid_update.get_updated_rate(TIME) == 0
+    assert load.state._energy_requirement_Wh[TIME] == 25.0
+    load.event_tick()
+    assert load.bid_update.get_updated_rate(TIME) == 10
```

Comment:
Hmm, sorry to be relentless here. But this only checks for the functionality of `load.bid_update.get_updated_rate`, but not the correct bid placement. 
What happened to my suggestion, to access the `market.bids` member in the integrationtests?

Reply:
Ah, yes, you are right. And now I understand your unittest. Agreed.